custom/src/main/java/org/kiwix/kiwixmobile/custom/download/effects/FinishAndStartMain.kt

```diff
@@ -19,14 +19,23 @@
 package org.kiwix.kiwixmobile.custom.download.effects
 
 import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.findNavController
 import org.kiwix.kiwixmobile.core.base.SideEffect
 import org.kiwix.kiwixmobile.core.extensions.ActivityExtensions.start
+import org.kiwix.kiwixmobile.custom.R
+import org.kiwix.kiwixmobile.custom.download.pressBackCallback
 import org.kiwix.kiwixmobile.custom.main.CustomMainActivity
 import javax.inject.Inject
 
 class FinishAndStartMain @Inject constructor() : SideEffect<Unit> {
```

Comment:
this class should probably be rewritten to be "NavigateToCoreReader"

Reply:
Why _CoreReader_?

Reply:
a typo on my behalf

Reply:
[9831148](https://github.com/kiwix/kiwix-android/pull/2371/commits/98311485a10bca97bc0a4f56353b2575ecf7533a)

Reply:
[65aed5b](https://github.com/kiwix/kiwix-android/pull/2371/commits/65aed5b8110c0be8fcf4f1a08cac51fec56bdc51)